### ISSUE TITLE
Laravel 6.0 compatibility

### ISSUE THREAD

**Issue by tillkruss** [clarkeash/doorman#44]
I can't install the packge on Laravel 6.0:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove clarkeash/doorman v3.0.0
    - Conclusion: don't install laravel/framework v6.0.0
    - Installation request for clarkeash/doorman ^3.0 -> satisfiable by clarkeash/doorman[v3.0.0].
    - Conclusion: don't install laravel/framework 6.x-dev
    - clarkeash/doorman v3.0.0 requires laravel/framework ^5.6 -> satisfiable by laravel/framework[5.6.x-dev, 5.7.x-dev, 5.8.x-dev].
    - Can only install one of: laravel/framework[6.0.x-dev, 5.6.x-dev].
    - Can only install one of: laravel/framework[6.0.x-dev, 5.7.x-dev].
    - Can only install one of: laravel/framework[6.0.x-dev, 5.8.x-dev].
    - Installation request for laravel/framework ^6.0 -> satisfiable by laravel/framework[6.0.x-dev, 6.x-dev, v6.0.0].
```

**Comment by lucasdcrk** [comment]
Laravel 6.0 compatibility will be available in Doorman v4.0 (see https://github.com/clarkeash/doorman/commit/9870fcaba34aaeb1fc09c42e946b2280176e97f0)
But @clarkeash need to release it ^^

**Comment by clarkeash** [comment]
@lucasdcrk ive only just pushed it, im releasing it now.
